### DISCUSSION TITLE
Should I reinstall the packages I've installed through the package manager with mason?

### DISCUSSION THREAD

**Discussion by ghost** [mason-org/mason.nvim#480] [Q&A] [answered]
In Manjaro I've opened neovim after installing NvChad which uses Mason.nvim and it's asking me which language servers to install.

I've already installed [many of the options](https://git.disroot.org/hirrolot19/dotfiles/raw/branch/main/install/arch/pkglist) it offers through the package manager so I don't know if I should install them again with mason, or remove them with the package manager and then install them with mason or leave it as it is and ignore mason.

If I don't need to install them shouldn't they appear as installed?

**Comment by WhoIsSethDaniel** [comment, marked as answer]
> If I don't need to install them shouldn't they appear as installed?

`mason` is only aware of the things it has installed. 

It depends greatly on how you use the tools/formatters/servers you wish to install, but, in general, *I* find it easier to use `mason` over my package manager for these things.
